hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/event/remote/RemoteTenantAwareEventTest.java

```diff
@@ -29,47 +29,71 @@
 @Story("RemoteTenantAwareEvent Tests")
 public class RemoteTenantAwareEventTest extends AbstractRemoteEventTest {
 
+    private static final String TENANT_DEFAULT = "DEFAULT";
+
+    private static final String APPLICATION_ID_DEFAULT = "Node";
+
+    @Test
+    @Description("Verifies that a MultiActionEvent can be properly serialized and deserialized")
+    public void testMultiActionEvent() {
```

Comment:
rename to reflect serialization/deserialization

Reply:
It's more than serialization /deserialization, therefore I'm for keeping the names...